sklearn/tree/export.py

```diff
@@ -36,13 +42,52 @@ def export_graphviz(decision_tree, out_file="tree.dot", feature_names=None,
     out_file : file object or string, optional (default="tree.dot")
         Handle or name of the output file.
 
-    feature_names : list of strings, optional (default=None)
-        Names of each of the features.
-
     max_depth : int, optional (default=None)
         The maximum depth of the representation. If None, the tree is fully
         generated.
 
+    feature_names : list of strings, optional (default=None)
+        Names of each of the features.
+
+    class_names : list of strings, bool or None, optional (default=None)
+        Names of each of the target classes in ascending numerical order.
+        Only relevant for classification and not supported for multi-output.
+        If ``True``, shows a symbolic representation of the class name.
+
+    label : {'all', 'root', 'none'}, optional (default='all')
+        Whether to show informative labels for error metric, etc.
+        Options include 'all' to show at every node, 'root' to show only at
+        the top root node, or 'none' to not show at any node.
+
+    filled : bool, optional (default=False)
+        When set to ``True``, paint nodes to indicate majority class for
+        classification, extremity of values for regression, or purity of node
+        for multi-output.
+
+    leaves_parallel : bool, optional (default=False)
+        When set to ``True``, draw all leaf nodes at the bottom of the tree.
+
+    metric : bool, optional (default=True)
+        When set to ``True``, show the error metric at each node.
```

Comment:
I would not use "error metric", but "impurity" instead. The criterion evaluated at each node is not necessarily an "error metric".